fe/user/src/pages/MapPage/Panel/VenueDetail/RestInfo/ShowInfo/ShowList.tsx

```diff
@@ -129,10 +137,16 @@ const StyledShowListContentHeader = styled.div`
   }
 `;
 
-const StyledShowListItem = styled.div`
+const StyledShowListItem = styled.div<{ $active: boolean }>`
   display: flex;
   align-items: center;
   gap: 14px;
+  ${({ $active }) =>
+    $active
+      ? css`
+          background-color: #f2f2f2;
+        `
+      : ''};
```

Comment:
styled-componenet에 있는 styled는 css로 감싸지 않고 문자열로도 되던 것 같은데 emotion은 잘 모르겠네요 만약 된다면 아래처럼 수정해도 될 것 같아요
```suggestion
  ${({ $active }) => $active && `background-color: #f2f2f2;`}
```

Reply:
수정 완료 https://github.com/jazz-meet/jazz-meet/pull/289/commits/30531f30fcee29d4cc1c9bc796365c5f63c4a8d2